examples/mpi/cntk-mpi.json

```diff
@@ -5,9 +5,9 @@
   // prepare cmudict corpus in CNTK format https://git.io/vbT5A and upload to hdfs
   "dataDir": "$PAI_DEFAULT_FS_URI/examples/cntk/data",
   // make a new dir for output on hdfs
-  "outputDir": "$PAI_DEFAULT_FS_URI/examples/cntk/output",
+  "outputDir": "$PAI_DEFAULT_FS_URI/examples/mpi/cntk/output",
```

Comment:
if you have not fixed cntk, don't include it in this PR. I suggest this PR focuses on horovod only.

Reply:
This pr is for revising document and fix some bugs of these examples, the horovod pr is #1615.